File: v2/csharp/src/main/csharp/io/swagger/Model/Advsior.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Advsior {
    

    
    public string Id { get; set; }

    

    
    public string VestorlyAuth { get; set; }

    

    
    public bool? NewUser { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Advsior {\n");
      
      sb.Append("  Id: ").Append(Id).Append("\n");
      
      sb.Append("  VestorlyAuth: ").Append(VestorlyAuth).Append("\n");
      
      sb.Append("  NewUser: ").Append(NewUser).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/jaxrs/src/main/java/io/swagger/model/Advsior.java
package io.swagger.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Advsior  {
  
  private String id = null;
  private String vestorlyAuth = null;
  private Boolean newUser = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("vestorlyAuth")
  public String getVestorlyAuth() {
    return vestorlyAuth;
  }
  public void setVestorlyAuth(String vestorlyAuth) {
    this.vestorlyAuth = vestorlyAuth;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newUser")
  public Boolean getNewUser() {
    return newUser;
  }
  public void setNewUser(Boolean newUser) {
    this.newUser = newUser;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Advsior {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  vestorlyAuth: ").append(vestorlyAuth).append("\n");
    sb.append("  newUser: ").append(newUser).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
